app/helpers/route_overrides_helper.rb

```diff
@@ -0,0 +1,37 @@
+module RouteOverridesHelper
+  class << self
+    private
+
+    def map_path(bad, *args)
```

Comment:
Sure, but I have a question. In the original, `new_` was not included. Was there any reason for it?
```
['', 'new_', 'edit_'].each do |prefix|
```
And btw, there's a minor bug in the original XD